PSICover/PSICover.cs

```diff
@@ -233,6 +330,17 @@ public override string ToString ()
    static string sLastFile = "";
 }
 
+class Summary {

```

Comment:
You could use record

Reply:
![image](https://github.com/SasiTMM/PSI/assets/125174306/6a8afdce-b570-4c97-a616-9f1f34bec543)